Tests/PostgresNIOTests/New/Messages/ErrorResponseTests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  File.swift
+//
+//
+//  Created by Fabian Fett on 12.01.21.
+//
+
+import NIO
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIO
```